docs/security/tokens/overview.md

```diff
@@ -1,22 +1,186 @@
 DateReviewed: 2022-03-10
-
+=======
```

Comment:
```suggestion
```

I don't think this is actually necessary looking at our other docs with `DateReviewed` metadata